### DISCUSSION TITLE
Seleción estilo de la página

### DISCUSSION THREAD

**Discussion by tsmanuelanton** [Arquisoft/dede_es4b#59] [Show and tell]
[Link ](https://athemes.com/collections/best-css-frameworks/)para consultar frameworks de CSS. Votad! (Si quereís)

**Comment by uo271355** [comment]
Mis favoritos son:
-https://tailwindcss.com/
https://milligram.io/#getting-started

**Comment by tsmanuelanton** [comment]
Yo voto por [Tailwind ](https://tailwindcss.com/). Me gusta el diseño y tiene buena documentación.

**Reply by tsmanuelanton** [reply]
Además hay un plugin para vscode.

![image](https://user-images.githubusercontent.com/90830684/157251112-9f82f7f5-4bdd-431f-a712-4eeaf7a3f17e.png)

**Comment by uo277553** [comment]
Yo también voto por [Tailwind ](https://tailwindcss.com/), el diseño me gusta más y que tenga un plugin para vscode está bastante bien.

**Comment by uo277878** [comment]
A mí también me gusta más [Tailwind](https://tailwindcss.com/) porque tiene más ejemplos para entenderlo mejor, me parece más bonito y tiene más opciones, como por ejemplo, el modo oscuro.

**Comment by UO276208** [comment]
Tailwind está perfecto.

**Comment by tsmanuelanton** [comment]
Dicidido, ha ganado:
![Tailwind](https://user-images.githubusercontent.com/90830684/160012084-e161a518-8704-4861-b7fc-34f997bf9008.png)
